packages/data-provider/src/utils/auditUtils.ts

```diff
@@ -24,7 +19,7 @@ export const isAuditEventValid = (value: unknown): boolean =>
  * @returns {AuditAction | undefined}
  */
 export const convertToAuditEvent = (value: string): AuditAction | undefined => {
-	const parseResult = auditActionSchema.safeParse(value.toUpperCase());
+	const parseResult = AUDIT_ACTION.transform((action) => action.toUpperCase()).safeParse(value.toUpperCase());
```

Comment:
you are transforming the value before sending it to the parser, you don't need to add the transform!

Reply:
removed transform function as we're already passing the uppercase value